.github/workflows/postgis-check-command.yml

```diff
@@ -0,0 +1,126 @@
+name: postgis-check
+
+on:
+  workflow_dispatch:
+    inputs:
+      issue-number:
+        description: 'The issue-number of the slash command'
+        required: true
+      comment-creation-date:
+        description: 'The slash command creation date'
+        required: true
```

Comment:
can you improve the description so its easier to understand

Reply:
Added some text. At the end this is a parameter passed from the slash-command workflow.